onlyoffice

```diff
@@ -0,0 +1,13 @@
+[Section]
```

Comment:
File doesn't have .txt extension, so it's not added to RAPPS

Reply:
I fixed that with https://github.com/reactos/rapps-db/commit/238ebb848f89d1852985d19f0f51f79b2298825e
Thx for the mentioning @julenuri